examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,180 @@
+        self.gstc.pipeline_pause(self.pipeName)
+
+    def stopVideo(self):
+        print("Video stopped")
+        if (self.pipe_exists(self.pipeName)):
+            self.gstc.pipeline_stop(self.pipeName)
+            self.gstc.pipeline_delete(self.pipeName)
+            self.thErrorHandler.join()
+
+    def setSpeed(self, speed):
+        print("Setting play speed to: "+str(speed))
+        self.playingSpeed = speed
+        self.gstc.event_seek(self.pipeName,
+                             rate=self.playingSpeed,
+                             format=3,
+                             flags=1,
+                             start_type=1,
+                             start=0,
+                             end_type=1,
+                             end=-1)
+
+    def jumpTo(self, position):
+        print("Jump to time: "+str(position))
+        self.gstc.event_seek(self.pipeName,
+                             rate=float(self.playingSpeed),
+                             format=3,
+                             flags=1,
+                             start_type=1,
+                             start=(position*(1000000000)),
+                             end_type=0,
+                             end=-1)
+
+    def pipe_exists(self, pipe_name):
+        # Check if pipe is already created
+        existing_pipes = self.gstc.list_pipelines()
+        if (existing_pipes == []):
+            ret = False
+        elif(existing_pipes[0]['name'] == pipe_name):
+            ret = True
+        else:
+            ret = False
+        return ret
+
+
+def printUsage():
+    print("play: To play and run")
+    print("pause: To pause the video")
+    print("stop: To stop and close the playing")
+    print("set_speed $SPEED")
+    print("jump $TIME [in seconds]")
+
+
+def printError():
+    print("To use run: python3 simple_pipeline.py $VIDEO_PATH",
+          file=sys.stderr)
+
+
+if __name__ == "__main__":
+    print("Sample PyGstC Video Player")
+
+    if(len(sys.argv) != 2):
+        printError()
+        sys.exit(0)
+    try:
+        myPlayer = GstcPlayer()
+
+        if(not myPlayer.openVideo(sys.argv[1])):
+            print("Pipeline already exits", file=sys.stderr)
+            sys.exit(0)
+
+        action = ["running"]
+        while (action[0] != "exit"):
+
+            if (action[0] == "play"):
+                myPlayer.playVideo()
+            elif (action[0] == "pause"):
+                myPlayer.pauseVideo()
+            elif (action[0] == "stop"):
+                myPlayer.stopVideo()
+            elif (action[0] == "set_speed"):
+                if (len(action) == 2):
+                    myPlayer.setSpeed(float(action[1]))
+                else:
+                    print("Playback speed valid range: [-1, 1]. 0 is not \
+                           allowed")
+
+            elif (action[0] == "jump"):
+                if (len(action) == 2 and
+                   action[1].isnumeric() and
+                   int(action[1]) >= 0):
```

Comment:
This is not necessarily limited to integers.

Reply:
I think it is: 
`event_seek TypeError: parameter 5: expected <class 'int'>, '<class 'float'> found`